chainer/links/normalization/layer_normalization.py

```diff
@@ -0,0 +1,53 @@
+from chainer import initializers
+from chainer import link
+
+from chainer.functions.array import broadcast
+from chainer.functions.math import sqrt
+from chainer.functions.math import square
+from chainer.functions.math import sum
+from chainer.links.connection import scale
+
+
+class LayerNormalization(link.Chain):
+
+    """Layer normalization layer on outputs of linear functions.
+
+    This is a link of "Layer Normalization". This layer
```

Comment:
Consider rewording:

This link implements a "layer normalization" layer which normalizes the input units by statistics that are computed along the second axis, scales and shifts them with :class:`~chainer.links.Scale`.